contrib: Fix analyze_local_images when image has missing layers

josuesdiaz opened the pull request:
This issue was originally addressed in #69, but looks to have been
reverted since then. As of right now, certain images that have missing
layers or layers that are not in the docker save output will cause an error
with the analyze_local_images command. Switching back to reading layers
from manifest.json file from docker save of the image.

jzelinskie commented:
Hey there,

It looks like this is (sadly) the best way to do this. I had reproduced a bug locally using this method, but it seems far less likely to occur than to have the command have missing layers.

Can you make your commit simply a `git revert` of `e639367a3082c5a2067803e9c66f1258d4e2b919`?

Commit:
contrib: Revert "Merge pull request #367 from jzelinskie/analyze-layers-v2"

This reverts commit e639367a3082c5a2067803e9c66f1258d4e2b919, reversing
changes made to f3848d9726ead6a8c2fd5f85e45c3b35e9899979. Issue involved
the docker history command providing missing fs layers that were not in
the saved image. Revert commit to using the manifest file again.

josuesdiaz commented:
Ya sure, sorry should have thought to do that in the first place. Updated the branch, and you should see the revert only in PR now.

jzelinskie reviewed (approved):
LGTM.
